README.md

```diff
@@ -0,0 +1,19 @@
+# Intel Retail Documentation Repository
+
+![Requirements Review Process Flow](./requirements-review-process-flow.png)
+
+## Requirements Process Breakdown
+
+- Submit a github issue and follow the requirements template.
+
+- Once a week there will be a requirements review meeting where we will approve, reject, or ask for modification to the requirement.
```

Comment:
should it be once a week or once every(#) week ?

Reply:
I'll change that. Currently we meet once a week but that can change depending on what works for us.